doc/tutorials/isingmodel.ipynb

```diff
@@ -0,0 +1,612 @@
+{
```

Comment:
<p>Here and elsewhere: Python beauty standard has a space before comma</p><p><br></p><p>circuit3(var1, var2)</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>